source/_integrations/zha.markdown

```diff
@@ -20,8 +20,8 @@ ha_codeowners:
   - '@adminiuga'
 ---
 
-[Zigbee Home Automation](https://zigbeealliance.org)
-integration for Home Assistant allows you to connect many off-the-shelf Zigbee based devices to Home Assistant, using one of the available Zigbee radio modules that is compatible with [zigpy](https://github.com/zigpy/zigpy) (an open source Python library implementing a Zigbee stack, which in turn relies on separate libraries which can each interface a with Zigbee radio module a different manufacturer).
+[Zigbee Home Automation (ZHA)](https://www.home-assistant.io/integrations/zha/)
```

Comment:
It is now linking to itself?

Reply:
I don't think that the title of the integration should itself link to zigbeealliance.org since the implementation of not an official Zigbee Alliance integration. Maybe otherwise just remove that link completely?

Reply:
Yeah, that sounds better. Linking to the page you are already at, is a bit useless :)